modules/javafx.controls/src/main/java/javafx/scene/control/Cell.java

```diff
@@ -611,12 +612,12 @@ public void cancelEdit() {
      *
      * <p>In general there is no need to override this method in custom cell
      * implementations - it should be sufficient to simply call this method
-     * when appropriate (e.g. when the user pressed the Enter key, you may do something
+     * when appropriate (e.g., when the user pressed the Enter key, you may do something
```

Comment:
Since you are changing this, perhaps you might also fix the one in the previous paragraph as well -- the one in the `<ol>` ... `</ol>` block?